lib/x11regressiontest.pm

```diff
@@ -111,7 +112,7 @@ sub cleanup_libreoffice_recent_file {
 }
 
 sub open_libreoffice_options {
-    if (is_tumbleweed) {
+    if (is_tumbleweed or sle_version_at_least('15')) {
```

Comment:
If you want to test a SLE version you also need to use `is_sle`.

Reply:
Fixed, please help to review.